common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -414,6 +414,16 @@ object CometConf extends ShimCometConf {
     .doubleConf
     .createWithDefault(10.0)
 
+  val COMET_EXCHANGE_SIZE_MULTIPLIER: ConfigEntry[Double] = conf(
+    "spark.comet.shuffle.sizeInBytesMultiplier")
+    .doc(
+      "Comet produces smaller shuffle files due to columnar compression and this can result in " +
```

Comment:
So ther eis no way to get the uncompressed size at the moment?

Reply:
We are using the uncompressed size of the Arrow batches, using this field in `shuffle_writer`:

```rust
    /// The original size of spilled data. Different to `spilled_bytes` because of compression.
    data_size: Count,
```

This is the code that updates it:

```rust
        // Update data size metric
        self.metrics.data_size.add(input.get_array_memory_size());
```

Reply:
I pushed a commit to update the config description to reflect this and to remove the mention of compression.